department_app/tests/test_rest.py

```diff
@@ -0,0 +1,240 @@
+"""test rest api"""
```

Comment:
Please split the test cases for departments and employees into 2 different modules.

Reply:
done